networktocode/pyntc
# prep 0.20.3 release (#254)

* update full project to new dev standards (#250)

* update full project to new dev standards part 1

* remove old unused files

* remove old unused files

* disable pylint ci test temporarily

* disable pylint ci test temporarily

* initial new docs build out

* get new docs introduced via mkdocs

* initial few pylint fixes

* get pytest passing

* update all project dependencies and make sure tests still pass

* remove sphinx ci stage and add some needs clauses

* fix install mode and update changelog

* Apply suggestions from code review

Co-authored-by: Ken Celenza <ken@celenza.org>

* readthedocs config file and update changelogs/release notes

* add python stylizer in docs

* newer version in dunder init

* add logo

* fix mkdocs formatting

Co-authored-by: Ken Celenza <ken@celenza.org>

* fix library summary (#253)

* add meta to pyproject and add license (#255)

Co-authored-by: Ken Celenza <ken@celenza.org>
